### PR TITLE
MRG: MWW test

### PR DESCRIPTION
**Pull request by doutib**
Mann–Whitney–Wilcoxon (MWW) non parametric test on distances.

### CODE REVIEW

**Comment by doutib** [comment]
@hduongtrong do you have any suggestion for the tests I can do for this function? For example the first one pvalues which computes p values of hypothesis tests on a bunch of sample distribution.

**Comment by doutib** [comment]
@lunadadada @z357412526 @CenzhuoYao Do you have any suggestion for the code ? 
- Are the docstrings clear enough?
- Any idea to make it faster?
- What about writing style?

File: mousestyles/mww.py [hunk 1]
```diff
@@ -0,0 +1,231 @@
+from scipy.stats import mannwhitneyu
+import itertools
+import numpy as np
+import matplotlib.pyplot as plt
+
+from mousestyles.data import distances_bymouse, distances_bystrain
+
+
```
Comment (hduongtrong) [inline mousestyles/mww.py]: function name could be verb, e.g. get_pvalues?

File: mousestyles/mww.py [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+def MWW_mice(strain, step=50, verbose=False):
+    """
+    Compare distributions of distances among mice of the same strain.
+    Use p-values of the Mann–Whitney U test.
+
+    Parameters
+    ----------
+    strain: integer
+        Number of the strain.
+    step: time interval length used to compute distances. Default is 1s.
```
Comment (hduongtrong) [inline mousestyles/mww.py]: the format maybe is:
step: integer
    desctiption....

**Comment by hduongtrong** [comment]
@doutib yeah testing function maybe: generate 1000 standard Gaussian, generate another 1000 standard Gussian, perform the function and make sure the pvalues is > say 0.5? Then generate 1000 standard gaussian, generate 1000 standard exponential, then make sure the pvalues is < say 0.5?

**Comment by doutib** [comment]
@hduongtrong thank you for the review, I have just addressed them.
@lunadadada @z357412526 @CenzhuoYao do you have any idea of what could make the code better, and what other tests I can add for the other functions?

File: mousestyles/mww.py [hunk 3]
```diff
@@ -0,0 +1,234 @@
+# -*- coding: utf-8 -*-
+
```
Comment (hduongtrong) [inline mousestyles/mww.py]: should we have the 
`from __future__ import print_function, division, absolute_import` for python 3 support?, also I think the standard is import itertools first, then numpy, scipy, and other after

**Comment by doutib** [comment]
@jarrodmillman 
Are we able to merge this PR? It helps to perform the Mann–Whitney–Wilcoxon test over mice distances.

**Pull request merged**